script/testing/junit/src/TracefileTest.java

```diff
@@ -73,6 +72,7 @@ public void setUp() throws FileNotFoundException, SQLException {
                     dTest.add(cur);
                 }
                 catch (Throwable e) {
+                    System.out.println(mog.sql);
```

Comment:
Is this print statement superfluous?

Reply:
Sorry it isn't supposed to be there. It was added back to the code when I rebased and was never run...